internal/armada/configuration/types.go

```diff
@@ -102,6 +102,8 @@ type SchedulingConfig struct {
 	MaximalResourceFractionPerQueue map[string]float64
 	// Max number of jobs to scheduler per lease jobs call.
 	MaximumJobsToSchedule uint
+	// Max number of gangs to scheduler per lease jobs call.
+	MaximumGangsToSchedule uint
```

Comment:
Why do we have this?  Is it because we don't want to schedule too may pods at once?